apps/projects/app/components/Panel/ReviewApplication/ReviewApplication.js

```diff
@@ -34,12 +36,32 @@ const ReviewApplication = ({ issue, requestIndex, readOnly }) => {
   } = useAragonApi()
   const { closePanel } = usePanelManagement()
   const theme = useTheme()
-
   const [ feedback, setFeedback ] = useState('')
-  const [ index, setIndex ] = useState(requestIndex)
 
-  const updateFeedback = e => setFeedback(e.target.value)
+  const canReview = issue.workStatus !== 'fulfilled' && !readOnly
+
+  const typesToShow = []
+  if (issue.requestsData.findIndex(request => !('review' in request)) !== -1)
+    typesToShow.push(canReview ? 'Available for review' : 'Unreviewed')
+  if (issue.requestsData.findIndex(request => 'review' in request) !== -1)
+    typesToShow.push('Reviewed')
```

Comment:
> Do you find it more readable this way?

Yes, thx!

> why do we say 'Unreviewed' if it's available for review

Because design says so - under some circumstances, if panel is opened in RO mode, applications are not allowed to be reviewed, and marking them as "Available" would be misleading...